doc/source/schema/schema.md

```diff
@@ -79,6 +122,11 @@ additive constant).
 the accepted proposal.
 * `max_energy_error`: The maximum absolute difference in Hamiltonian energy between the initial point and all possible samples in the proposed tree.
 * `int_time`: The total integration time (static HMC sampler)
+* `mass_matrix`: Mass matrix (also known as _metric_) used in HMC samplers for the computation of the Hamiltonian.
```

Comment:
Isn't it more common to represent the inverse of the metric instead of the metric itself? I believe this is what Stan does, and this is also what Julia's HMC samplers do. In many/most cases the inverse metric is diagonal, so perhaps we should explicitly say that diagonal metrics can be represented as just the diagonal?

Personally I prefer the term metric over mass matrix.

Reply:
Stan also prefers metric (mass_matrix comes from physics)

Reply:
I'll update this then, it looks like using `inv_metric` would be best right? also feel free to suggest better definitions or editing it directly